pkg/api/service/util.go

```diff
@@ -104,45 +87,16 @@ func NeedsHealthCheck(service *api.Service) bool {
 
 // GetServiceHealthCheckNodePort Return health check node port for service, if one exists
 func GetServiceHealthCheckNodePort(service *api.Service) int32 {
```

Comment:
Agreed. I was also thinking to remove these helpers when I first hit them. As you said, they're not needed anymore. Will address this. Thanks!